server/mldb_server.h

```diff
@@ -131,6 +131,7 @@ struct MldbServer: public ServicePeer, public EventRecorder {
     std::string getCacheDirectory() const;
 
     std::string httpBoundAddress;
+    std::string httpBaseUrl;
```

Comment:
I would not keep this variable in the server unless we need it in the future.  I would pass it to the init function of the server and pass it to the static content handler.